### PR TITLE
CachingUtilities.py: Fix description

### PR DESCRIPTION
**Pull request by SubbulakshmiRS**
Fix description for pickle_dump and pickle_load 
functions.

Fixes https://github.com/coala/coala/issues/5442

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by SubbulakshmiRS** [comment]
I have changed the number of commits into 1 . Also appveyor which is failing seems to be something about windows ! Please help  @jayvdb

**Comment by jayvdb** [comment]
Windows problem has been fixed.

File: coalib/misc/CachingUtilities.py [hunk 1]
```diff
@@ -68,8 +68,12 @@ def delete_files(log_printer, identifiers):
 
 def pickle_load(log_printer, identifier, fallback=None):
     """
-    Get the data stored in ``filename`` present in the user
-    config directory. Example usage:
+    ``identifier`` represents a file whose path is got using
```
Comment (jayvdb) [inline coalib/misc/CachingUtilities.py]: This isnt a valid docstring.  See PEP 257

**Comment by SubbulakshmiRS** [comment]
CI has passed .

**Comment by jayvdb** [comment]
Something is wrong with your commit, or your GitHub user profile, as https://github.com/coala/coala/pull/5741/commits/39105b3d8157a4831b70bc0a75ffe5fa6edc6b8f doesnt link to your username

**Comment by SubbulakshmiRS** [comment]
Sorry for the trouble , my local github profile had not been properly configured (must have changed when my laptop crashed a few weeks back ! ) It is correct now I think ! Please check .

**Comment by SubbulakshmiRS** [comment]
The extra line was there to separate between example and the summary line .
The above function get_data_path in the same file has the filename in double quotes and thus I followed suit .
 I removed the file location because it was already mentioned below , where the arguments of the function is explained . 
Should I still make those changes requested ? @rishabhgarg25699

**Comment by rishabhgarg25699** [comment]
OK @SubbulakshmiRS maintainers will check this .

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5741/commits/3d617e8b545cd03b0f723d3d947ed0b763b94827

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**